lib/ModuleFilenameHelpers.js

```diff
@@ -47,11 +49,11 @@ function asRegExp(test) {
 }
 
 ModuleFilenameHelpers.createFilename = function createFilename(module, moduleFilenameTemplate, requestShortener) {
-	var absoluteResourcePath;
-	var hash;
-	var identifier;
-	var moduleId;
-	var shortIdentifier;
+	let absoluteResourcePath;
+	let hash;
+	let identifier;
+	let moduleId;
+	let shortIdentifier;
 	if(!module) module = "";
```

Comment:
Would it be safe to convert this to `if (module === undefined) module = "";`?

That would make future conversion to argument defaults (`function createFilename(module = "", /* ... */) { /* ... */ }`) trivial.